pom.xml

```diff
@@ -44,6 +44,12 @@ Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
 	<properties>
 	<org.exoplatform.social.version>6.2.x-maintenance-SNAPSHOT</org.exoplatform.social.version>
     <org.exoplatform.platform-ui.version>6.2.x-maintenance-SNAPSHOT</org.exoplatform.platform-ui.version>
+
+
+    <!-- Maven Plugin for JS and CSS bundling and Unit Test execution -->
+    <com.github.eirslett.frontend.version>1.6</com.github.eirslett.frontend.version>
+    <node.version>v12.1.0</node.version>
+    <npm.version>6.9.0</npm.version>
```

Comment:
Please undo these changes. This must be handled by the maven-parent-pom module.

Reply:
@nidhalnidhal please take care of this

Reply:
actually the build of the module is failing when i restored the previous versions of node and npm.